DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: ospsuite
 Title: R package to manipulate OSPSuite Models
-Version: 12.0.0.9000
+Version: 12.0
```

Comment:
we should keep the version AS IS, the weird problem on AppVeyor has gone

Reply:
Cool; good to know that the issue is not the package version.